HomeTask_11.py

```diff
@@ -0,0 +1,23 @@
+"""HomeTask11.
+
+write generator function that has as input some range value and chunk_size
+produces output as lists with len == chunk size
+Example:
+Call:  chunk_generator(range(11), chunk_size=3) ->
+Output:  [0, 1, 2]
+         [3, 4, 5]
+         [6, 7, 8]
+         [9, 10]
+"""
+def chunk_generator(range_values, chunk_size):  # create a generator
+    range_list = list(range_values)  # create list
```

Comment:
Range also allows to do slices thus better to produce list at the end. This will save a little bit resources
```
def chunk_generator(range_values, chunk_size): 
    range_ = range_values
    for i in range(0, len(range_), chunk_size): 
        yield list(range_[i:i+chunk_size]) 
```